skins/oasis/css/modules/ContributeModule.scss

```diff
@@ -0,0 +1,46 @@
+@import 'skins/shared/mixins/flexbox';
+
+.contribute-module {
```

Comment:
We could polish this module when right rail is dropped to the bottom of the page. Currently tally and buttons are displayed one under another...